kv/local_test_cluster.go

```diff
@@ -146,5 +122,8 @@ func (ltc *LocalTestCluster) Start(t util.Tester) {
 
 // Stop stops the cluster.
 func (ltc *LocalTestCluster) Stop() {
+	if r := recover(); r != nil {
```

Comment:
thanks for the heads up. It did solve some crashes for me - should I leave it in?